### PR TITLE
quick{emu,get}: add page

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 4.9.2

Support for macOS hosts is experimental.

### CODE REVIEW

File: pages/linux/quickget.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- Download configuration for a Windows 11 VM with VirtIO drivers for Windows:
+
+`quickget windows 11`
+
+- Download a macOS recovery image and creates a virtual machine configuration:
+
+`quickget macos {{high-sierra|mojave|catalina|big-sur|monterey|ventura}}`
+
+- Show an ISO URL for an OS (do not work for Windows):
```
Comment (gutjuri) [inline pages/linux/quickget.md:24]:
```suggestion
- Show an ISO URL for an OS (does not work for Windows):
```

File: pages/linux/quickget.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+`quickget --show-iso-url fedora {{release}} {{edition}}`
+
+- Test if and OS ISO is available for an OS:
```
Comment (gutjuri) [inline pages/linux/quickget.md:28]:
```suggestion
- Test if an OS ISO is available for an OS:
```

File: pages/linux/quickget.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+`quickget --test-iso-url nixos {{edition}} {{plasma5}}`
+
+- Open an OS distribution homepage in a browser (does not work for Windows):
```
Comment (gutjuri) [inline pages/linux/quickget.md:32]:
```suggestion
- Open an OS distribution's homepage in a browser (does not work for Windows):
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions :)

**Review by gutjuri — APPROVED** [review]

File: pages/linux/quickemu.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Create and run a VM from a configuration file:
+
+`quickemu --vm {{path/to/file.conf}}`
+
+- Do not commit any changes to disk/snapshot but write any changes to temporary files:
+
+`quickemu --status-quo --vm {{path/to/file.conf}}`
+
+- Start the VM in full screen mode (<Ctrl> + <Alt> + f to exit) and select the display backend ('sdl' by default):
```
Comment (kbdharun) [inline pages/linux/quickemu.md:15]:
```suggestion
> See also: `quickget`, for preparing VM configurations.
> More information: <https://github.com/quickemu-project/quickemu>.

- Create and run a virtual machine from a configuration file:

`quickemu --vm {{path/to/file.conf}}`

- Do not commit any changes to disk/snapshot but write any changes to temporary files:

`quickemu --status-quo --vm {{path/to/file.conf}}`

- Start the virtual machine in full-screen mode (<Ctrl> + <Alt> + f to exit) and select the display backend ('sdl' by default):
```

File: pages/linux/quickemu.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`quickemu --fullscreen --display {{sdl|gtk|spice|spice-app|none}} --vm {{path/to/file.conf}}`
+
+- Select a virtual audio emulation and create a desktop shortcut:
```
Comment (kbdharun) [inline pages/linux/quickemu.md:19]:
```suggestion
- Select a virtual audio emulation device and create a desktop shortcut:
```
Don't know if it's the correct term here, feel free to reword it.

File: pages/linux/quickget.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# quickget
+
+> Download and prepare materials for building a Quickemu virtual machine.
+> NOTE: the parameter "edition" is always optional.
+> See also: `quickemu`.
+> More information: <https://github.com/quickemu-project/quickemu>.
+
+- Display the list of all supported guest OSes, versions and variants:
```
Comment (kbdharun) [inline pages/linux/quickget.md:8]:
```suggestion
- Display the list of all supported guest operating systems, versions and variants:
```

File: pages/linux/quickget.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+- Download configuration for a Windows 11 VM with VirtIO drivers for Windows:
+
+`quickget windows 11`
+
+- Download a macOS recovery image and creates a virtual machine configuration:
+
+`quickget macos {{high-sierra|mojave|catalina|big-sur|monterey|ventura}}`
+
+- Show an ISO URL for an OS (does not work for Windows):
```
Comment (kbdharun) [inline pages/linux/quickget.md:24]:
```suggestion
- Show an ISO URL for an operating system (Note: it does not work for Windows):
```

File: pages/linux/quickget.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+`quickget --show-iso-url fedora {{release}} {{edition}}`
+
+- Test if an OS ISO is available for an OS:
```
Comment (kbdharun) [inline pages/linux/quickget.md:28]:
```suggestion
- Test if an ISO file is available for an operating system:
```

File: pages/linux/quickget.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+`quickget --test-iso-url nixos {{edition}} {{plasma5}}`
+
+- Open an OS distribution's homepage in a browser (does not work for Windows):
```
Comment (kbdharun) [inline pages/linux/quickget.md:32]:
```suggestion
- Open an operating system distribution's homepage in a browser (Note: it does not work for Windows):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/quickemu.md (hunk 5 above)
Comment (kbdharun) [reply pages/linux/quickemu.md:19]:
```suggestion
- Select virtual audio emulation and create a desktop shortcut:
```
Maybe this?

Comment (vitorhcl) [reply pages/linux/quickemu.md:19]:
```suggestion
- Select a virtual audio device to emulate and create a desktop shortcut:
```
@kbdharun what about this?

Comment (kbdharun) [reply pages/linux/quickemu.md:19]: Sure, LGTM.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**